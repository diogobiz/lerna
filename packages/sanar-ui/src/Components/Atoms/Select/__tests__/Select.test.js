import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESSelect, { ESOption } from '../'

it('renders correctly', () => {
    const component = (
        <ESSelect style={{ width: 200 }} placeholder='Select a person'>
            <ESOption value='jack'>Jack</ESOption>
            <ESOption value='lucy'>Lucy</ESOption>
            <ESOption value='tom'>Tom</ESOption>
        </ESSelect>
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
