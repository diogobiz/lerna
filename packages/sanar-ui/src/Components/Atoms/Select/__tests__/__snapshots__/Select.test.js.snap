// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="es-select ant-select ant-select-enabled"
  onBlur={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onMouseDown={[Function]}
  onMouseOut={[Function]}
  onMouseUp={[Function]}
  onTouchStart={[Function]}
  style={
    Object {
      "width": 200,
    }
  }
>
  <div
    aria-autocomplete="list"
    aria-controls="test-uuid"
    aria-expanded={false}
    aria-haspopup="true"
    className="ant-select-selection
            ant-select-selection--single"
    onKeyDown={[Function]}
    role="combobox"
    tabIndex={0}
  >
    <div
      className="ant-select-selection__rendered"
    >
      <div
        className="ant-select-selection__placeholder"
        onClick={[Function]}
        onMouseDown={[Function]}
        style={
          Object {
            "WebkitUserSelect": "none",
            "display": "block",
            "userSelect": "none",
          }
        }
        unselectable="on"
      >
        Select a person
      </div>
    </div>
    <span
      className="ant-select-arrow"
      onClick={[Function]}
      style={
        Object {
          "WebkitUserSelect": "none",
          "userSelect": "none",
        }
      }
      unselectable="on"
    >
      <i
        className="anticon es-eva-icon ant-select-arrow-icon es-eva-icon--medium"
      >
        <i
          data-eva="chevron-down-outline"
          data-eva-fill="#fff"
          data-eva-height="18px"
          data-eva-width="18px"
        />
      </i>
    </span>
  </div>
</div>
`;
